packages/reporting/src/markdown/index.mjs

```diff
@@ -1,2 +1,2 @@
-export { Table } from "./Table";
+export { Table } from "./table.mjs";
```

Comment:
This import was incorrect and was causing the [tests to fail](https://github.com/aws-powertools/actions/actions/runs/8969046670/job/24629688481).

I think we didn't notice before due to our local OS not being case-sensitive.

While here I also added the file extension.

Reply:
another TIL. Expected it to fail locally too. Thank you for the fix!